zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/HttpZipkinFactory.java

```diff
@@ -36,6 +36,7 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(HttpZipkinFactory.class);
 
   @NotEmpty private String baseUrl = "http://127.0.0.1:9411/";
+  @NotEmpty private String spansAPIRelativePath = "api/v2/spans";
```

Comment:
strongly prefer not doing this as it is two items of configuration one of which not used elsewhere. for example, baseUrl and endpoint are used elsewhere, but the relative path isn't. we'd have to define this concept only here...

I'd suggest prioritizing endpoint over baseUrl, then forming endpoint from baseUrl when not around.